app/scripts/components/common/card/uswds-cards/styles.scss

```
@@ -0,0 +1,9 @@
+@use '$styles/veda-ui-theme-vars.scss' as themeVars;
+
+.usa-card {
+  margin-bottom: 0;
```

We need to remove the left and right margin as well, to have the `<a>` tag match the card itself. How about 
```suggestion
  margin: 0;
```

---

Ah wait, the left and right margin are set on the `usa-card__container`, not the usa-card itself.